SDK/MobileCenter/Microsoft.Azure.Mobile.iOS/MobileCenter.cs

```diff
@@ -174,10 +174,14 @@ private static Class[] GetServices(IEnumerable<Type> services)
             var classes = new List<Class>();
             foreach (Type t in services)
             {
-                if((GetBindingType(t) != null) && (GetClassForType(GetBindingType(t)) != null))
+                var bindingType = GetBindingType(t);
+                if((bindingType != null) && (GetClassForType(bindingType) != null))
```

Comment:
I think `&& (GetClassForType(bindingType) != null)` can be removed as you check again right after